Fail to build OCIS with docker

## Describe the bug

Fail to build OCIS using docker.

## Steps to reproduce

Steps to reproduce the behavior:

1. Clone or download the repository
2. Open terminal and digit: docker build -t ocis .

## Expected behavior
No erros on building the application using docker.

## Actual behavior
A message of error on building web and graph-explorer, missing the assets directory.

#16 236.3 git clean -xfd assets
#16 236.4 Removing assets/
#16 236.4 curl --fail -slL -o- https://github.com/owncloud/web/releases/download/v5.7.0-rc.8/web.tar.gz | tar xzf - -C assets/
#16 237.7 tar: assets: Cannot open: No such file or directory
#16 237.7 tar: Error is not recoverable: exiting now
#16 237.7 make[2]: *** [Makefile:36: pull-assets] Error 2
#16 237.7 make[1]: *** [Makefile:158: ci-node-generate] Error 1
#16 237.7 make: *** [Makefile:42: ci-node-generate] Error 2

## Additional context
Correction:
Insert "mkdir assets", before the curl command, in
\ocis\services\graph-explorer\Makefile, and
\ocis\services\web\Makefile

--- a/Dockerfile
+++ b/Dockerfile
@@ -21,14 +21,14 @@ COPY ./ /ocis/
 WORKDIR /ocis/ocis
 RUN make ci-node-generate
 
-FROM owncloudci/golang:1.17 as build
+FROM owncloudci/golang:1.18 as build
 
 COPY --from=generate /ocis /ocis
 
 WORKDIR /ocis/ocis
 RUN make ci-go-generate build
 
-FROM alpine:3.13
+FROM alpine:3.15
 
 RUN apk update && \
 	apk upgrade && \

